### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by dkryvi**

### CODE REVIEW

File: app/utils/initializer.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
 
-// init logger, db connection etc.
 module.exports.appReady = () => Promise.all([
-  // dbConnection, connect to storage
-  loggerInitialization(),
-]);
+  elasticSearchInitialization(),
+])
+  .then(() => {
+    logger.info('Successfully initialize all services');
```
Comment (akosenko) [inline app/utils/initializer.js]: All services initialized successfully

Comment (dkryvi) [reply app/utils/initializer.js]: Ok. I'll fix

Comment (dkryvi) [reply app/utils/initializer.js]: [Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/13/commits/7e10bfd671ee954c3b1eef19d05ccb6b0e9c31a4#diff-96a8661bff3c7b9cefe5405b208b2830R24)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+# Event Emitter backend
+
+## Testing
+
+Before test please run:
+
+`docker-compose -f docker-compose.test.yml up`
+
+This command creates elastic search container.
+You ay connect to Elasticsearch through 127.0.0.1:9200 or 127.0.0.1:9300.
```
Comment (Vanish404) [inline README.md]: I think you made a mistake at this sentence
`ay`

File: app/utils/initializer.js [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+const { elasticSearchClient } = require('../services/elasticSearch');
+
+const elasticSearchInitialization = () => new Promise((resolve, reject) => {
+  elasticSearchClient.ping({
+    requestTimeout: 1000
+  }, (err) => {
+    if (err) {
+      logger.error(`elasticsearch connection error: ${err}`);
+      reject(err);
+    } else {
```
Comment (Vanish404) [inline app/utils/initializer.js]: in my opinion, i should say that we can refuse `else` statement, you can `return` one of callback functions. i'm talking about `resolve` and `reject`

**Review by Vanish404 — CHANGES_REQUESTED** [review]

File: app/utils/initializer.js [hunk 4]
```diff
@@ -1,10 +1,28 @@
 require('../config');
-require('../services/elasticSearch');
 
-const loggerInitialization = () => Promise.resolve();
+const logger = require('../utils/logger')('initializer');
+const { elasticSearchClient } = require('../services/elasticSearch');
+
+const elasticSearchInitialization = () => new Promise((resolve, reject) => {
+  elasticSearchClient.ping({
```
Comment (Vanish404) [inline app/utils/initializer.js:7]: I think, you need to `return` this method

**Review by Vanish404 — CHANGES_REQUESTED** [review]

File: app/utils/initializer.js (hunk 4 above)
Comment (dkryvi) [reply app/utils/initializer.js:7]: Why?

File: README.md (hunk 2 above)
Comment (dkryvi) [reply README.md]: [Fixed](https://github.com/inst-event-emitter/eventemitter-backend/pull/13/commits/03b617748894f02a8f8d8baa3f39a267c570be3d#diff-04c6e90faac2675aa89e2176d2eec7d8R12)

File: app/utils/initializer.js (hunk 3 above)
Comment (dkryvi) [reply app/utils/initializer.js]: Yeap. I [update](https://github.com/inst-event-emitter/eventemitter-backend/pull/13/commits/03b617748894f02a8f8d8baa3f39a267c570be3d#diff-96a8661bff3c7b9cefe5405b208b2830R14) this code

File: app/utils/initializer.js (hunk 4 above)
Comment (Vanish404) [reply app/utils/initializer.js:7]: I read documentation about `Promise`, i'm sorry, i was wrong

**Review by Vanish404 — APPROVED** [review]

**Pull request merged**